src/ExplainParser.php

```
@@ -267,8 +267,8 @@ private function parseSingleTable(array $table): TreeNode
     {
         /** @var TreeNodeAttributes $attributes */
         $attributes = [
-            'rows' => (string) $table['rows_examined_per_scan'],
-            'filtered' => (string) $table['filtered'],
+            'rows' => isset($table['rows_examined_per_scan']) ? (string) $table['rows_examined_per_scan'] : null,
```

PHPStan (2.1.33)

Offset 'rows_examined_per…' on array{table_name: string, access_type: string, possible_keys?: string|null, key?: string|null, rows: int, rows_examined_per_scan: int, filtered: float, attached_condition?: string, ...} in isset() always exists and is not nullable.


(isset.offset)